src/css/information.scss

```diff
@@ -15,112 +15,124 @@
   .quick-id {
     color: $white;
     background: $navy;
+    float: left;
 
-    h2,
-    table {
-      width: 310px;
-    }
-
-    h2 {
-      color: $white;
-    }
+    .content {
+      position: relative;
+      top: 10px;
+      left: 60%;
```

Comment:
Not ideal. Your layout will be broken on smaller screen sizes.

If you need that big gap on the left, maybe you just need a small px value gap on right.